sputnikdao2/src/upgrade.rs

```diff
@@ -9,6 +10,7 @@ const DEFAULT_FACTORY_ID: &str = "sputnik-dao.near";
 const ERR_MUST_BE_SELF_OR_FACTORY: &str = "ERR_MUST_BE_SELF_OR_FACTORY";
 const SELF_MIGRATE_METHOD_NAME: &[u8; 7] = b"migrate";
 const UPDATE_GAS_LEFTOVER: Gas = Gas(5_000_000_000_000);
+const GAS_LEFTOVER: Gas = Gas(15_000_000_000_000);
```

Comment:
Nit pick - Can you change to more concise name here? `FACTORY_UPDATE_GAS_LEFTOVER` perhaps?

Reply:
sure, good suggestion